docs/dev/konnector.md

```diff
@@ -1,39 +1,38 @@
-# How to write a collector
+# How to write a connector
 
 ## Introduction
 
-A collector is a small script that allow to import data from an external website. Each collector is an independent application, managed by the Cozy Collect application.
+A connector is a small script that allows to import data from an external website. Each connector is an independent application, managed by the Cozy Collect application.
 
-To protect your data, each collector runs inside a sandbox where all their interactions with your data are under control.
+To protect your data, each connector runs inside a sandbox where all their interactions with your data are under control.
 
-From a technical point of view, collectors are Node.js applications executed inside a container. They communicate with the server using its API, like client side apps, and get an auth token every time they start. They need to register with a manifest, and ask permissions to the user.
+From a technical point of view, connectors are Node.js applications executed inside a container. They communicate with the server using its API, like client side apps, and get an auth token every time they start. They need to register with a manifest, and ask permissions to the user.
 
-To ease the development of a collector, the environment inside the container provides a lot of shared libraries. You don’t need to include this libraries with your script, they will always been available. Here’s a list:
+To ease the development of a connector, an npm package, named [cozy-connector-libs](https://github.com/cozy/cozy-konnector-libs) provides a lot of
+shared libraries. But you may need some other npm packages to help you running your connector: 
 
- - [cozy-connector-libs](https://github.com/cozy/cozy-konnector-libs) : collectors specific shared functions ;
- - [cozy-client.js](https://github.com/cozy/cozy-client-js) an abstraction layer to use the Cozy API ;
```

Comment:
Could you please list the libraries embedded into cozy-konnector-libs and give an example of how to call them? (`require('cozy-konnector-libs').cozyClient`)